### DISCUSSION TITLE
Updating College Email Address for GitHub Campus Expert Status After College Transfe

### DISCUSSION THREAD

**Discussion by mishba-ai** [community/community#65056] [GitHub Education]
### Hi everyone,

I transferred to a new college for specific reasons and now I wish to update my college email address associated with my GitHub Campus Expert status because in GitHub student developer pack it is showing my ex-university name. Could you please guide me on how to proceed with this?

I have less time as I want to apply for GitHub campus expert programme and deadline is sep 1,2023

plz help me with this

**Comment by mishba-ai** [comment]
Thank you,but I am stuck at point 3
could you share the screenshot where I can find that?
and yeah the university I am in does not provide college Id can you use my personal email Id

**Comment by lab596** [comment]
My apologies the previous instructions were incorrect, 

1. Log into GitHub

2. Once logged in, navigate to your GitHub settings by clicking on your profile picture in the top-right corner of the page. Then, click on "Settings" in the dropdown menu.

3. Then locate "Emails" under the Access label and add your new email under "Add email address" (this can be a non school email)

4. Verify your email by checking the email inbox provided and signing into your github account

5. Go to https://education.github.com/ and sign into your account

6. Once logged in, click on your profile picture in the top-right corner of the page. Then, click on "Reverify your academic affiliation" in the dropdown menu.

7. Select your new email and provide all required information for reverification

It may take a few days for the verification.

Sorry for the previous oversight, let me know if this helps!

**Comment by mishba-ai** [comment]
I am not able to reverify it, I also want to change the university name, and after changing the email id,it shows my previous university name
and you can see
![image](https://github.com/community/community/assets/121809535/63646fc4-794c-4251-8287-1ad394c38be1)

**Comment by lab596** [comment]
Each college has its own requirements, putting the college name should display any specifications for the email that had to be met. Then that email had to be added to the github profile according to the previous steps. The error you may be facing is using a school issued email for a different university.

**Comment by mishba-ai** [comment]
How can I overcome this issue? anything else we can do?

**Comment by lab596** [comment]
I dont understand the error you are facing. Is the email you provided not allowing you to continue with the reverification? An error message related to the email should be provided. For ex. it will say that the email must include a "@student.edu" email.

**Reply by mishba-ai** [reply]
![image](https://github.com/community/community/assets/121809535/1b2ceda1-ce27-4181-8b3b-63aa7a6d70ec)

**Comment by lab596** [comment]
Hey I believe I found a work around that is better than creating a new GitHub account:

If you go to GitHub settings and go under `Billing and plans`>`Plans and usage` you should be able to disable GitHub pro by clicking Edit>Downgrade to Free

From there you should go back to your GitHub education website and sign up for GitHub Pro using your updated information

**Reply by mishba-ai** [reply]
Let me give it a try, well Thank you for this and
for signing up with the new email or the same it was previously?

**Comment by lab596** [comment]
It can be any email, by adding and verifying a new email like mentioned in the previous steps you are able to add a new email or you can use the same one you have already been using.

**Reply by mishba-ai** [reply]
if I downgrade to free then after signing up will I get my old repos and achievements back?

**Comment by FAIZUAN55** [comment]
> ### Hi everyone,
> 
> I transferred to a new college for specific reasons and now I wish to update my college email address associated with my GitHub Campus Expert status because in GitHub student developer pack it is showing my ex-university name. Could you please guide me on how to proceed with this?
> 
> I have less time as I want to apply for GitHub campus expert programme and deadline is sep 1,2023
> 
> plz help me with this
> 
> ### A suggestion...(optional)
> 
> _No response_

Sorry

**Reply by mishba-ai** [reply]
why are you sorry? I don't understand

**Reply by FAIZUAN55** [reply]
Sorry for sending by mistake
